k8s/common-base/neuvector/fluentbit.yaml

```diff
@@ -0,0 +1,32 @@
+---
+apiVersion: flux.weave.works/v1beta1
+kind: HelmRelease
+metadata:
+  name: fluentbit-log
+  namespace: neuvector
+  annotations:
+    flux.weave.works/automated: "false"
+spec:
+  releaseName: fluentbit-log
+  chart:
+    repository: https://hmcts.azurecr.io/helm/v1/repo/
```

Comment:
we should republish this to hmctspublic... need a separate story though probably we forgot about this :p